### DISCUSSION TITLE
Import

### DISCUSSION THREAD

**Discussion by Soffan76** [pimcore/pimcore#17611] [General]
Hi,

I am new at Pimcore and I´m wondering about importing new values to descriptors from an Excel-file. Is there a documentation somewhere of this? I have learned how to do an export to create an Excel-file, but then I want to import it again after modification and I don´t understand how.

**Comment by developkariyer** [comment]
You can use DataHub (https://pimcore.com/docs/platform/Datahub/) with many options available or you can write a PHP API script, which parses CSV (exported from Excel) and update relevant objects in Pimcore.

**Comment by dpfaffenbauer** [comment]
You could also use Data Definitions which supports excel out of the box already: https://github.com/instride-ch/pimcore-data-definitions

**Comment by Soffan76** [comment]
Ok, thank you both for your answers, I will try to look into it.

In previous PIM systems I have worked with it seems much more simple.
1. You click an import button
2. A wizard appears to take you through the steps with mapping etc.
3. Different choises of import, if you just want to to a test import for example
4. Import your new data
